src/pipecat/serializers/livekit.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+import ctypes
+import pickle
+
+from livekit.rtc import AudioFrame
```

Comment:
We should add a new `livekit` option in pyproject.toml. And here we should check if `livekit.rtc` is available otherwise show an error like in other processors.